### PR TITLE
Add pagination for dashboard

### PR DESCRIPTION
**Pull request by aviskarkc10**

### CODE REVIEW

File: src/pages/Dashboard.js [hunk 1]
```diff
@@ -286,85 +300,124 @@ const DashBoardComponent = () => {
...
-          : searchText?.length > 0 && (
-              <NoPatientsWrapper>
-                <p>
-                  <strong>No results found</strong>
-                </p>
-                <Button onClick={clearSearchInput} className="link-button">
-                  Back to dashboard
-                </Button>
-              </NoPatientsWrapper>
+      {isMobile ? (
```
Comment (nastikue) [inline src/pages/Dashboard.js]: Please make the same way it is done in Patients.js: move mobile and desktop code in separate components and leave in return clear condition.
image

File: src/components/common/Layout/DashboardLayout/index.js [hunk 2]
```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import * as PropTypes from 'prop-types';
 import CallDialog from 'components/common/CallDialog';
 
-const DashboardLayout = ({ children }) => {
+const DashboardLayout = ({ children, style }) => {
   return (
     <>
       <main>
-        <div className="content">{children}</div>
+        <div className={`content ${style}`}>{children}</div>
```
Comment (nastikue) [inline src/components/common/Layout/DashboardLayout/index.js]: please rename style to customClass or any name with 'class' in it

File: src/pages/Dashboard.js [hunk 3]
```diff
@@ -286,85 +300,124 @@ const DashBoardComponent = () => {
...
+                  <>Load More</>
+                )}
+              </Button>
+            </div>
+          ) : null}
+        </PatientsWrapper>
+      ) : (
+        <DeskTopViewPatient isInitLoading={isInitLoading}>
+          <HeaderSearchWrap className="w-100 mb-3">
+            {!isLightVersion && (
```
Comment (nastikue) [inline src/pages/Dashboard.js]: I think we can get rid of isLightVersion everywhere

**Review by nastikue — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard.js (hunk 3 above)
Comment (aviskarkc10) [reply src/pages/Dashboard.js]: this might be required in the future so will leave it as it is for now

**Review by nastikue — APPROVED** [review]

**Pull request merged**